flow/layers/shader_mask_layer_unittests.cc

```diff
@@ -98,8 +98,9 @@ TEST_F(ShaderMaskLayerTest, SimpleFilter) {
   const SkPaint child_paint = SkPaint(SkColors::kYellow);
   auto layer_filter =
       SkPerlinNoiseShader::MakeFractalNoise(1.0f, 1.0f, 1, 1.0f);
+  auto dl_filter = DlColorSource::From(layer_filter);
```

Comment:
Eeek. Someone created these random SkShader objects for the tests here that represent nothing we'd ever see in practice and we aren't testing that "we do the right thing with a Fractal Noise shader", we're testing that "we do the right thing with *a* shader (any shader)". So, we should really just change these to instantiate some token Dl...ColorSource directly and stay away from creating an SkShader and using the ::From() converter. If we don't do that now, we'll eventually do it when we go through and get rid of Skia from the source base so rather than *add* a usage of ::From() here, switch to a natively constructed Dl*ColorSource, of any type.

Reply:
... in all of the unit tests ...

Reply:
And all of the blend mode values should just be `s/Sk/Dl/` if we switch to that type for SML construction...

Reply:
These tests largely assert against a mock canvas object that derives from Skia. I have a slight preference for fixing this up in a follow on patch. WDYT?

Reply:
The Skia object is obsolete and randomly obscure and completely irrelevant (type-wise) to the test. I've been suggesting replacing rather than adding a new line of code to wrap them wherever we are updating the unit tests.

Reply:
Hmm, it would complicate the MockCanvas validation at the end of the unit test. I see that issue now. For new unit tests I've been pushing a migration to the new DL comparison method of checking the output, but for existing unit tests that's a bit of a haul.

Point conceded.

But we should update the layer to take a DlBlendMode...

Reply:
Done